graylog2-web-interface/src/components/permissions/EntityShareModal.tsx

```diff
@@ -52,7 +53,9 @@ const EntityShareModal = ({ description, entityId, entityType, entityTitle, enti
   const _handleSave = () => {
     setDisableSubmit(true);
     const granteesSelect = granteesSelectRef?.current;
+    // @ts-ignore We dealt with the undefined case here
     const granteesSelectValue = granteesSelect?.state?.value;
+    // @ts-ignore We dealt with the undefined case here
```

Comment:
I am still curious why TypeScript complains about the optional chaining here. It looks like we can remove the `@ts-ignore`s here, with `const granteesSelectRef = useRef<typeof Select>();`